README.md

```diff
@@ -13,14 +13,14 @@ jobs:
     steps:
       - uses: AButler/upload-release-assets@v1.0
```

Comment:
while you're in here, can you update this to the current version?
https://github.com/AButler/upload-release-assets/issues/3

Reply:
updated